specification/app/Microsoft.App.DynamicSessions/models/code-execution.tsp

```diff
@@ -0,0 +1,72 @@
+import "./common.tsp";
+using Azure.Core;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+
+namespace Microsoft.App.DynamicSessions;
+
+@doc("The request to execute code.")
+model SessionCodeExecutionRequest {
+  @doc("The code input type of the code execution request.")
+  codeInputType: CodeInputType;
+
+  @doc("The execution type of the code execution request.")
+  executionType: ExecutionType;
+
+  @doc("The string of the code to execute, based on CodeInputType.")
+  code: string;
+
+  @doc("Code execution timeout in seconds.")
+  timeoutInSeconds: int64;
+}
+
+@doc("The response containing the result of the code execution.")
+model SessionCodeExecutionResponse {
+  @visibility("read")
+  @doc("Session code execution id.")
+  executionId?: string;
+
+  @doc("The status of the code execution operation, indicates whether succeeded or not.")
+  status: Foundations.OperationState;
+
+  @doc("The standard output of the code execution.")
+  stdout?: string;
```

Comment:
error etc. should follow LRO response structure

Reply:
Error field is in the auto-generated spec

Reply:
Resolved